Formula/android-ndk.rb

```diff
@@ -1,9 +1,9 @@
 class AndroidNdk < Formula
-  desc "Android native-code language toolset"
+  desc "Android native-code language toolset (Native Development Kit)"
   homepage "https://developer.android.com/ndk/index.html"
-  url "https://dl.google.com/android/repository/android-ndk-r12-darwin-x86_64.zip"
-  version "r12"
-  sha256 "fe10b64a65f3e818b852862975ea7b5896295cd1301d1acdbeb1294d9592fb19"
+  url "http://dl.google.com/android/repository/android-ndk-r12b-darwin-x86_64.zip"
```

Comment:
Please don't downgrade to `http://` here. Should still be able to use `https://`.